tests/unit/test.linter.js

```diff
@@ -1501,15 +1501,18 @@ describe('Linter.extractMetadata()', () => {
       code: 'COINMINER_USAGE_DETECTED',
       column: 284,
       line: 2,
-      dataPath: 'CoinHive.CONFIG',
+      // TODO: determine why with ajv 8 dataPath or instancePath
+      // are not set to these values, and if we actually care
+      // (given that the validation warning is added anyway).
+      // instancePath: 'CoinHive.CONFIG',
```

Comment:
This is probably caused by a missing change in `src/linter.js`. We need to update `dataPath` to `instancePath`:

```diff
diff --git a/src/linter.js b/src/linter.js
index 804c663f..9f6df5a5 100644
--- a/src/linter.js
+++ b/src/linter.js
@@ -758,9 +758,9 @@ export default class Linter {
             file: filename,
             line: matchedLine,
             column: matchedColumn,
-            // use dataPath for our actual match to avoid any obvious
+            // use instancePath for our actual match to avoid any obvious
             // duplicates
-            dataPath: match[0],
+            instancePath: match[0],
           });
         }
       });
```

Reply:
thank you! I looked to linter but at that point my brain got tired of looking for dataPath and filtered this out without telling me :-)

I'm going to apply this change and confirm that the related test case does pass as expected (and remove the TODO on the test side if that is the case as it seems it should be).